src/index.html

```diff
@@ -1,13 +1,66 @@
 <!doctype html>
-<html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
+<html lang="en" class="card-page">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
```

Comment:
send attribute to the next line

Reply:
why? if there is only one attribute you can do it in the same line;